scripts/test.js

```diff
@@ -44,7 +44,7 @@ const createOption = ({ defaultValue, option, name, extraParam }) => ({
 
 const tasks = {
   core: createProject({
-    name: `Core & React & Vue & Polymer & Angular & Svelte ${chalk.gray('(core)')}`,
+    name: `Core & React & Vue & Polymer & Angular & Svelte & Riot ${chalk.gray('(core)')}`,
```

Comment:
Let's maybe call this `Core & Examples` ? 🤷‍♂️

Reply:
Good idea